src/Middleware/RequestThrottling/src/RequestThrottlingOptions.cs

```diff
@@ -15,5 +15,11 @@ public class RequestThrottlingOptions
         /// This is null by default because the correct value is application specific. This option must be configured by the application.
         /// </summary>
         public int? MaxConcurrentRequests { get; set; }
+
+        /// <summary>
+        /// Maximum number of queued requests before the server starts rejecting connections.
+        /// The server will respond with a 503 if this limit is exceeeded.
+        /// </summary>
+        public int RequestQueueLimit { get; set; } = 5000;
```

Comment:
5000 * number of CPUs

Reply:
The queue limit was fixed at 5000 the old one, so I'm just copying that. The number of concurrently running requests was 5000*CPUs

Reply:
Add a comment in a remarks sections below summary mentioning that the limit was copied from IIS. Maybe include https://blogs.msdn.microsoft.com/benjaminperkins/2011/11/14/modify-the-request-queue-limit-requestqueuelimit-or-queue-length-in-iis-7/ or a similar doc as a reference.

Reply:
> Add a comment in a remarks sections below summary mentioning that the limit was copied from IIS. Maybe include https://blogs.msdn.microsoft.com/benjaminperkins/2011/11/14/modify-the-request-queue-limit-requestqueuelimit-or-queue-length-in-iis-7/ or a similar doc as a reference.

Not needed. That's interesting to us but nobody else. A code comment would be sufficient.

Reply:
Sure. @DylanDmitri here is an example: https://github.com/aspnet/AspNetCore/blob/master/src/Servers/Kestrel/Core/src/KestrelServerLimits.cs#L27